### PR TITLE
Ensure we suggest the deprecated attribute in bridge deprecation warnings

### PR DESCRIPTION
**Pull request by stack72**
Fixes: #442

### CODE REVIEW

**Review by guineveresaenger — CHANGES_REQUESTED** [review]
Since we have [some test coverage ](https://github.com/pulumi/pulumi-terraform-bridge/blob/master/pkg/tfshim/tfplugin5/diagnostics_test.go), we should add a test case for diagnostic Details.

Otherwise LGTM.

**Comment by guineveresaenger** [comment]
I'm confused - can we not add a `Detail` field to [this line]https://github.com/pulumi/pulumi-terraform-bridge/blob/master/pkg/tfshim/tfplugin5/diagnostics_test.go#L14 and test for that?

If that test file does not map to [pkg/tfshim/tfplugin5/diagnostics.go](https://github.com/pulumi/pulumi-terraform-bridge/pull/562/files#diff-f07ee4eef978964a5b3d3491ee005c8a3ec4cf4974bdc8df0156ba8cfa4aff00) then please disregard.

File: pkg/tfshim/sdk-v2/diagnostics.go [hunk 1]
```diff
@@ -14,7 +14,11 @@ func warningsAndErrors(diags diag.Diagnostics) ([]string, []error) {
 		case diag.Error:
 			errors = append(errors, fromV2Diag(d))
 		case diag.Warning:
-			warnings = append(warnings, d.Summary)
+			details := d.Summary
```
Comment (jkodroff) [inline pkg/tfshim/sdk-v2/diagnostics.go:17]: Can we add a comment citing an example of what might be in `Summary` vs `Detail` and why we would would prefer the latter?

File: pkg/tfshim/tfplugin5/diagnostics.go [hunk 2]
```diff
@@ -18,7 +18,11 @@ func unmarshalWarningsAndErrors(diags []*proto.Diagnostic) ([]string, []error) {
 		case proto.Diagnostic_ERROR:
 			errors = append(errors, fromTF5ProtoDiag(d))
 		case proto.Diagnostic_WARNING:
-			warnings = append(warnings, d.Summary)
+			details := d.Summary
```
Comment (jkodroff) [inline pkg/tfshim/tfplugin5/diagnostics.go:21]: Same as above.

**Review by jkodroff — CHANGES_REQUESTED** [review]

File: pkg/tfshim/sdk-v2/diagnostics.go (hunk 1 above)
Comment (stack72) [reply pkg/tfshim/sdk-v2/diagnostics.go:17]: So the fix kinda denotes that right? I mean, the summary doesn't include the parameter name, the detail does :)

**Review by jkodroff — APPROVED** [review]

**Pull request merged**